shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -30,6 +30,7 @@ - (void)updateEditingState;
 - (BOOL)isVisibleToAutofill;
 - (id<FlutterTextInputDelegate>)textInputDelegate;
 - (void)configureWithDictionary:(NSDictionary*)configuration;
+- (void)replaceRangeLocal:(NSRange)range withText:(NSString*)text;
```

Comment:
Why can't the behavior be tested via the text input protocol, rather than internal implementation details?